src/lib/migrations/routines/04-update-message-updates.ts

```diff
@@ -150,7 +150,7 @@ const updateMessageUpdates: Migration = {
 	_id: new ObjectId("5f9f4f4f4f4f4f4f4f4f4f4f"),
```

Comment:
don't we have to change the id? 
Otherwise, since migration with this id was already recorded as done, it will not run. No?

Reply:
or should we create new migration `src/lib/migrations/routines/06-xyz.ts` ?

Reply:
Yeah good call, updated the id so it will run again. The first run didn't work so this should be safe